wn/_add.py

```diff
@@ -73,6 +74,7 @@
 def add(
     source: AnyPath,
     progress_handler: Optional[Type[ProgressHandler]] = ProgressBar,
+    lmf_load_progress: Optional[Type[ProgressHandler]] = ProgressBar,
```

Comment:
The pattern I used for `wn.download()` is that the value of `progress_handler` is passed to `wn.add()`. I think the same could be done here. That means the *class* and not the instance is passed along.